src/System.Diagnostics.Process/src/System/Diagnostics/ThreadInfo.cs

```diff
@@ -12,7 +12,7 @@ namespace System.Diagnostics
     /// <internalonly/>
     internal sealed class ThreadInfo
     {
-        internal int _threadId;
+        internal ulong _threadId;
```

Comment:
What's the benefit of changing this to be a ulong instead of an int?  I'd understand it if storing it as a ulong meant we could then use that ID elsewhere more appropriately on OSX (e.g. passing it to some function that needed the 64-bit value), but that doesn't appear to be the case.  As a result, we're increasing the amount of space required and we're having to cast in more places, but I'm not seeing the benefit...?